src/components/pages/ParentContact/ParentContactContainer.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import { Layout, Row, Col } from 'antd';
+import img2 from './img2.png';
+import ParentNavTopBar from '../../common/ParentNavTopBar';
+
+const ParentContactContainer = () => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [message, setMessage] = useState('');
+  const [item, setItem] = useState([]);
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    setItem([...item, { name: name, email: email, message: message }]);
+    setName('');
+    setEmail('');
+    setMessage('');
+  };
+
+  useEffect(() => {
+    const data = localStorage.getItem('data');
+    if (data) {
+      setItem(JSON.parse(data));
+    }
+  }, []);
+
+  useEffect(() => {
+    localStorage.setItem('data', JSON.stringify(item));
+  });
+  return (
+    <>
+      <Layout className="parent-dashboard">
+        <ParentNavTopBar />
+        <div className='"site-card-border-less-wrapper"'>
+          <Row className="card">
+            <Col span={16}>
+              <Row className="title">
+                <h4 style={{ color: '#878D93', fontWeight: '700' }}>
+                  CONTACT US
+                </h4>
+              </Row>
+              <div className="description">
+                <Row>
+                  <p>
+                    If you would like to let us know how we're doing of if you
+                    are experiencing issues, contact us by filling in the form
+                    bellow.
+                  </p>
+                </Row>
+              </div>
+            </Col>
+            <Col
+              span={8}
+              type="flex"
+              style={{ alignItems: 'center' }}
+              justify="bottom"
+            >
+              <img
+                style={{ position: 'absolute', overflow: 'scroll' }}
+                alt="example"
+                src={img2}
+              />
+            </Col>
+          </Row>
+        </div>
+        <div className="form-main">
+          <Row className="card">
+            <form className="contact-form-container" onSubmit={handleSubmit}>
+              <Row className="form-inputs">
```

Comment:
Looks like you are using ant-d for everything except the form itself? What was the reasoning behind this technical decision?